### DISCUSSION TITLE
Sandik problemi

### DISCUSSION THREAD

**Discussion by illkerrr** [IBAX-io/go-ibax#1425] [Soru Cevap - Türkçe]
Nerdeyse Her actigim sandıkta 0 puan alıyorum. Başka başına gelen oldumu? Yoksa bu bir açıkmı???

**Comment by Yusufknk** [comment]
Sandığa bir kere tıklayın. Birden fazla kez tıklandığında ilk çıkan puan gösterilmediği için 0 olarak gösterir.

**Reply by ibaxmustafaceviz** [reply]
bana da sıfır puan veriyor bir kez tıklıyorum.

**Reply by Yusufknk** [reply]
Sandıktan 0 çıkabilir. Üste üste 2 defa da olabilir, ama her zaman sıfır çıkmaz.

**Reply by illkerrr** [reply]
10 kere sandik açtıysam 5inde 0 puan verdi
